src/IonTextReader.ts

```diff
@@ -318,6 +318,25 @@ export class TextReader implements Reader {
     throw new Error("Current value is not a blob or clob.");
   }
 
+  uInt8ArrayValue(): Uint8Array | null {
+    this.load_raw();
```

Comment:
Rather than maintaining two copies of this function body, could you modify `byteValue()` to call `uInt8ArrayValue()` as its implementation?